src/systemtask/WakeLock.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "systemtask/SystemTask.h"
+
+namespace Pinetime {
+  namespace System {
+    class WakeLock {
+    public:
+      WakeLock(SystemTask& systemTask);
+      void Lock();
+      void Release();
+
+      ~WakeLock() {
+        Release();
+      }
+
+    private:
+      bool lockHeld;
+      SystemTask& systemTask;
+    };
+  }
+}
```

Comment:
add newline at end of file to prevent stop symbol of GitHub 
![image](https://github.com/user-attachments/assets/0195b16f-7b5b-458f-92b4-5b76d82ed75a)

```suggestion
}

```

Reply:
Good spot, I'll fix this after feedback on the other suggestion thread

Should (can?) we change the autoformatter to enforce this? I thought it caught this kind of thing already

Reply:
If it is possible with `clang-format` then yes, I'd love it if it is enforced. But as `clang-format` is a black magic box for me I don't know how. I'm always happy for projects where somebody did put in the work to set-up `clang-format`

Reply:
I've just checked, and the option to insert trailing newlines has been added in clang-format 16. Last time I updated it was 14, so it wasn't included in that. I'll do a upgrade to a newer version soon.

What version do you think is reasonable to update to? Unfortunately, I don't think that clang-format ignores unrecognised options, so it'll be a minimum version for anyone using it.

Reply:
Opened https://github.com/InfiniTimeOrg/InfiniTime/issues/2127 to continue this conversation